presto-main/src/main/java/com/facebook/presto/operator/project/PageProcessor.java

```diff
@@ -221,21 +282,21 @@ public long getRetainedSizeInBytes()
                 }
 
                 updateRetainedSize();
-                return Optional.of(page);
+                return ofResult(page);
             }
         }
 
         private void updateRetainedSize()
         {
             // increment the size only when it is the first reference
-            retainedSizeInBytes = Page.INSTANCE_SIZE + SizeOf.sizeOfObjectArray(page.getChannelCount());
+            AtomicLong retainedSizeInBytes = new AtomicLong(Page.INSTANCE_SIZE + SizeOf.sizeOfObjectArray(page.getChannelCount()));
```

Comment:
Why local variable became an `AtomicLong`?

Reply:
I don't see what you are talking about in the code below.